di_website/templates/workforus/work_for_us_page.html

```diff
@@ -0,0 +1,98 @@
+
+{% block hero %}
+    {% include 'includes/heroes/hero.html' %}
+{% endblock hero %}
+
+{% block body %}
+
+    <section class="section">
+        <div class="row row--narrow">
+            <div class="is-typeset is-typeset--additionals max-meter">
+                {% for block in page.body %}
+                    {% if block.block_type == 'paragraph_block' or block.block_type == 'block_quote' %}
+                        {% include_block block %}
+                    {% endif %}
+                {% endfor %}
+
+                <p>
+                {% if page.recruitment_policy %}
+                    View our <a href="{{ page.recruitment_policy }}">Recruitment Policy</a>
+                {% endif %}
+
+                {% if page.gdr_policy %}
+                    and <a href="{{ page.gdr_policy }}">GDPR policy</a></p>
+                {% endif %}
+            </div>
+        </div>
+    </section>
+
+    <section class="section">
+        <div class="row row--narrow">
+            <h2 class="section__heading">Benefits of working at DI</h2>
+            <div class="l-sidebar">
+                <div class="l-sidebar__body">
+                    {% for block in page.benefits %}
+                        {% with block.value as val %}
+                            {% include "includes/partials/benefits.html" %}
+                        {% endwith %}
+                    {% endfor %}
+                </div>
+                <aside class="l-sidebar__aside">
+                    {% for block in page.logos %}
+                        {% with block.value as item %}
+                            {% include "workforus/work_for_us_logos.html" %}
+                        {% endwith %}
+                    {% endfor %}
+                </aside>
+            </div>
+        </div>
+    </section>
+
+    <section class="section">
+        <div class="row row--narrow">
+            <h2 class="section__heading">Our values</h2>
+            <div class="is-typeset">
+                <p>{{ page.values_text }}</p>
+            </div>
+            <div class="l-3up">
+                {% values %}
+            </div>
+        </div>
+    </section>
+
+    <section class="section section--alt">
+        <div class="row row--narrow">
+            <h2 class="section__heading">Team stories</h2>
+            <div class="is-typeset max-meter">
+                <p>{{ page.team_story_heading }}</p>
+            </div>
+            <div class="l-4up space-section-top">
+                {% for block in page.team_story %}
+                    {% with block.value as story %}
+                        {% include "workforus/work_for_us_team_stories.html" %}
+                    {% endwith %}
+                {% endfor %}
+            </div>
+        </div>
+    </section>
+
+    <section class="section" id="vacancies">
+        <div class="row row--narrow">
+            {% if vacancies %}
+                <h2 class="section__heading">Latest vacancies</h2>
```

Comment:
I'd expect `page.vacancy_heading` to be here.

Reply:
Updated with a new variable `vacancy_title`